src/App.jsx

```diff
@@ -160,6 +177,18 @@ function App() {
                   {darkMode ? <FaSun className="text-black" /> : <FaMoon className="text-white" />}
                 </div>
 
+                {/* Save Drawing Button */}
+                <div className="flex justify-center items-center">
+                  <button
+                    className={`bg-[#CBCCCF] scale-[0.7] p-[1rem] text-[1.5rem] w-80% rounded-[50%] shadow-black shadow-md transform transition duration-300 ease-in-out text-black hover:bg-gray-400 cursor-pointer dark:bg-slate-800 dark:text-[#ffffff] hover:md:scale-[0.8] ${
+                      !showMenuAndBgColor && "mt-10"
+                    }`}
+                    onClick={saveDrawing}
+                  >
+                  <FaDownload size={24} />
+                  </button>
```

Comment:
Hi there @Bhumika1312 👋🏻 

Currently the `button` element is not accessible due to lack of attributes like `aria-label` & `title`. Since you're only using Icon's for Download button you also need to ensure to create an accessible solution.

Example:
```diff
<button
  className={`bg-[#CBCCCF] scale-[0.7] p-[1rem] text-[1.5rem] w-80% rounded-[50%] shadow-black shadow-md transform transition duration-300 ease-in-out text-black hover:bg-gray-400 cursor-pointer dark:bg-slate-800 dark:text-[#ffffff] hover:md:scale-[0.8] ${
    !showMenuAndBgColor && "mt-10"
  }`}
  onClick={saveDrawing}
+ aria-label='Upload Drawing to Google Drive'
+ title='Upload'
 >
  <FaDownload size={24} />
</button>
```

Reply:
Made the changes Please review it
Thank you